src/platform/x86_64/boot_comp.c

```diff
@@ -0,0 +1,359 @@
+#include "assert.h"
```

Comment:
Very hard to review this file. Unclear what is changed from the previous version.

Reply:
I will see if I could try use a soft link here for this file.

Reply:
fixed, shared it with both i386 and x86_64